analytic_operating_unit/static/src/components/analytic_distribution.esm.js

```diff
@@ -16,7 +21,7 @@ patch(AnalyticDistribution.prototype, {
                             [
                                 "operating_unit_ids",
                                 "in",
-                                this.props.record.data.operating_unit_id[0],
+                                this.props.record.data.operating_unit_id.id,
```

Comment:
Considering the code is being rewritten in the runtime, I would make this change safer.
Yep, OWL is used more and more and we should be able to access the `id` of an `operating_unit_id` object.
But, there is still some legacy code, I think, so I would go for:
```
const ou = this.props.record.data.operating_unit_id;
const ouId = ou?.id || ou?.[0];
```

Reply:
When testing, the value of `this.props.record.data.operating_unit_id[0]` was now undefined in version 19.0.
That's why I replaced it by `.id`.